packages/astro/src/client/index.ts

```diff
@@ -14,7 +17,10 @@ export const createClerkInstance = runOnce(createClerkInstanceInternal);
 export async function createClerkInstanceInternal(options?: AstroClerkIntegrationParams) {
   let clerkJSInstance = window.Clerk;
   if (!clerkJSInstance) {
-    await waitForClerkScript();
+    await loadClerkJsScript({
+      clerkJSVersion: HARDCODED_LATEST_CLERK_JS_VERSION,
+      ...(options as any),
```

Comment:
The `as any` is a code smell, ideally the `options` has the correct type, could be a footgun in the future